### PR TITLE
change interview date

### PR DESCRIPTION
**Pull request by ChornayaDrakoshig**

### CODE REVIEW

File: client/app/components/pages/employee-career-day-page/EmployeeCareerDayPage.tsx [hunk 1]
```diff
@@ -131,12 +162,13 @@ class EmployeeCareerDayPage extends React.Component<IProps, IState> {
     ));
   }
 
-  private isActiveAddObjectiveButton() {
+  ///////////// решить какое название оставить
+  private isArchived() {
     if (this.props.selectedCareerDay) {
       return this.props.selectedCareerDay.Archived;
+    } else {
```
Comment (TsvAliaksandra) [inline client/app/components/pages/employee-career-day-page/EmployeeCareerDayPage.tsx]: unnessesary `else`

File: client/app/components/pages/employee-career-day-page/EmployeeCareerDayPage.tsx [hunk 2]
```diff
@@ -172,11 +204,47 @@ class EmployeeCareerDayPage extends React.Component<IProps, IState> {
     return true;
   }
 
+  private renderDatetime() {
```
Comment (TsvAliaksandra) [inline client/app/components/pages/employee-career-day-page/EmployeeCareerDayPage.tsx]: move it in separate file

File: client/app/components/pages/employee-career-day-page/EmployeeCareerDayPage.tsx [hunk 3]
```diff
@@ -189,38 +257,85 @@ class EmployeeCareerDayPage extends React.Component<IProps, IState> {
             <Grid container justify="flex-end" className={classes.navigation}>
               <Grid item>
                 <Grid container spacing={8}>
+                  {this.state.datetimePopupIsOpen && (
```
Comment (TsvAliaksandra) [inline client/app/components/pages/employee-career-day-page/EmployeeCareerDayPage.tsx]: move ot to end of the render component

File: server/Domain/Services/Impl/ObjectiveService.ts [hunk 4]
```diff
@@ -21,8 +21,8 @@ export class ObjectiveService implements IObjectiveService {
     this._careerDayRepository = careerDayRepository;
   }
 
-  public async getObjectivesByCareerDayId(CareerDayId: number): Promise<CareerDayEntity[]> {
-    return this._careerDayRepository.findAll({
+  public async getObjectivesByCareerDayId(CareerDayId: number): Promise<CareerDayEntity> {
```
Comment (CeQadrat) [inline server/Domain/Services/Impl/ObjectiveService.ts]: Why "getObjectivesByCareerDayId" returns CareerDayEntity?
Maybe the method should be called something like "getCareerDayById"?

**Pull request merged**